### PR TITLE
Update .gitmodules

### PR DESCRIPTION
**Pull request by l20031018**
change url of lwIP

### CODE REVIEW

**Comment by iabdalkader** [comment]
@l20031018 Why do we need to change it ?

**Comment by l20031018** [comment]
> @l20031018 Why do we need to change it ?

I have some trouble with clone lwip.

File: .gitmodules [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	url = https://github.com/atgreen/libffi
```
Comment (sarrahtrembl) [inline .gitmodules:7]: Micro python

**Review by sarrahtrembl — APPROVED** [review]
￼￼￼✌️

**Comment by iabdalkader** [comment]
Going to switch to github mirror here https://github.com/lwip-tcpip/lwip

**Pull request closed without merging**